src/PowerSystem/PowerSystem.jl

```diff
@@ -0,0 +1,70 @@
+module PowerSystem
+
+using Printf
+using SparseArrays
+using ..ExaPF: ParsePSSE, ParseMAT, IndexSet, Spmat
+
+import Base: show
+
+const PQ_BUS_TYPE = 1
+const PV_BUS_TYPE = 2
+const REF_BUS_TYPE  = 3
+
+abstract type AbstractNetworkAttribute end
+struct NumberOfBuses <: AbstractNetworkAttribute end
+struct NumberOfLines <: AbstractNetworkAttribute end
+struct NumberOfGenerators <: AbstractNetworkAttribute end
+struct NumberOfPVBuses <: AbstractNetworkAttribute end
+struct NumberOfPQBuses <: AbstractNetworkAttribute end
+struct NumberOfSlackBuses <: AbstractNetworkAttribute end
+
```

Comment:
So, my understanding is that these structs are used to specify the "get" call. So, e.g., if I do

`get(pf, NumberOfBuses)`

then Julia knows that I am referring to:

`get(pf::PowerNetwork, ::NumberOfBuses) = pf.nbus`

Reply:
that's correct!